frontend/src/components/homepage/PreviewModal.tsx

```diff
@@ -86,36 +98,42 @@ const PreviewModal = ({
       isOpen={preview}
       onClose={previewBook}
       motionPreset="slideInBottom"
-      size="4xl"
+      size="6xl"
     >
       <ModalOverlay />
-      <ModalContent>
+      <ModalContent paddingLeft="20px">
         <ModalCloseButton position="static" marginLeft="auto" />
         <ModalHeader paddingTop="-10px">
-          <Heading as="h3" size="md">
+          <Heading as="h3" size="lg" marginBottom="10px" width="75%">
             {title}
           </Heading>
-          <Box padding="10px 0px 10px 0px" float="left">
-            <Badge>{`Level ${level}`}</Badge>
-            <Badge>{`${convertLanguageTitleCase(language)}`}</Badge>
-          </Box>
+          <Badge variant="language" size="s">{`${convertLanguageTitleCase(
+            language,
+          )}`}</Badge>
+          <Badge
+            variant={getLevelVariant(level)}
+            size="s"
+          >{`Level ${level}`}</Badge>
+          <Badge variant="stage" size="s">
+            {storyTranslationId ? "Review" : "Translation"}
+          </Badge>
           <Button
             float="right"
-            width="21%"
+            width="20%"
             colorScheme="blue"
-            mr={3}
+            size="secondary"
+            marginRight="15px"
             onClick={primaryBtnOnClick()}
           >
             {primaryBtnText}
           </Button>
         </ModalHeader>
-        <Box display="block" paddingLeft="30px">
-          <Text fontSize="16px" color="grey">
-            <Link href={youtubeLink} isExternal>
-              → Watch the English AniBook
-            </Link>
-          </Text>
-        </Box>
+        <ModalBody marginBottom="30px" marginTop="-10px">
+          <Link href={youtubeLink} isExternal>
```

Comment:
the link styling seems to be missing the underline+the text looks a bit darker than the figma mock
<img width="330" alt="image" src="https://user-images.githubusercontent.com/32009013/136697766-3a959fdf-aa48-44c3-80b7-18573dd2eee8.png">
compared to: 
<img width="349" alt="image" src="https://user-images.githubusercontent.com/32009013/136697779-2abd1b82-6d09-4999-bcc0-3cbe99fa4f82.png">